contracts/FairLaunchClaimer.sol

```diff
@@ -0,0 +1,172 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/security/Pausable.sol";
+import "@openzeppelin/contracts/access/AccessControl.sol";
+import "./AuctionSuperApp.sol";
+import "./GeoWebParcel.sol";
+import "./ERC721License.sol";
+import "./interfaces/IClaimer.sol";
+
+contract FairLaunchClaimer is Pausable, AccessControl, IClaimer {
+    bytes32 public constant PAUSE_ROLE = keccak256("PAUSE_ROLE");
+    bytes32 public constant CLAIM_ROLE = keccak256("CLAIM_ROLE");
+
+    /// @notice License
+    ERC721License public license;
+
+    /// @notice Parcel
+    GeoWebParcel public parcel;
+
+    /// @notice Auction
+    AuctionSuperApp public auctionApp;
+
+    /// @notice start time of the genesis land parcel auction.
+    uint256 public auctionStart;
+    /// @notice when the required bid amount reaches its minimum value.
+    uint256 public auctionEnd;
+    /// @notice start price of the genesis land auction. Decreases to endingBid between auctionStart and auctionEnd.
+    uint256 public startingBid;
+    /// @notice the final/minimum required bid reached and maintained at the end of the auction.
+    uint256 public endingBid;
+
+    /// @notice Emitted when a parcel is purchased
+    event ParcelPurchased(uint256 indexed parcelId, address indexed to);
```

Comment:
Should this event have "claim" in the name? Like `ParcelClaimed` or `ClaimedParcel`? Could be confused with someone purchasing an existing parcel from someone else.